src/lib/draw-and-pick.js

```diff
@@ -35,12 +35,26 @@ export function drawLayers({layers, pass}) {
       compositeCount++;
     } else if (layer.props.visible) {
 
+      const pickingSelectedColor = layer.selectedObjectPickingColor();
+      if (pickingSelectedColor) {
+        layer.state.model.updateModuleSettings({
+          pickingValid: true,
```

Comment:
This parameter name (`pickingValid`) does not feel intuitive to me. Would love to see an API audit to discuss what it does, why it is needed, and what possible alternatives could be.

But first: If we are making changes to the API of the picking module, we also need to update its docs, maybe copy them into this directly temporarily?

A thought: Most modules have a `...Enabled` param. This is different from that?